src/Lachain.Storage/State/StateManager.cs

```diff
@@ -21,8 +20,9 @@ public class StateManager : IStateManager
         private readonly ISnapshotManager<IValidatorSnapshot> _validatorManager;
 
         private readonly Mutex _globalMutex = new Mutex(false);
+        public IRocksDbContext _dbContext;
```

Comment:
should it be public ?

Reply:
Not sure if there is any problem with public, but keeping it private is better idea. Changing it now.